app/models/figure.rb

```diff
@@ -9,5 +10,12 @@ class Figure < ActiveRecord::Base
   validates :era, presence: true
   validates :claim_to_fame, presence: true
 
-
+  def average_rating
+    ratings = Rating.where(figure_id: id)
```

Comment:
@jennceng We chose this method for the figure model because we want to be able to call average_rating on an individual figure, we do not want to call average_rating on all figures